src/drivers/boards/px4fmu-v4/board_config.h

```diff
@@ -261,6 +261,9 @@ __BEGIN_DECLS
 #define GPIO_8266_PD			(GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_2MHz|GPIO_OUTPUT_SET|GPIO_PORTE|GPIO_PIN5)
 #define GPIO_8266_RST			(GPIO_OUTPUT|GPIO_PUSHPULL|GPIO_SPEED_2MHz|GPIO_OUTPUT_SET|GPIO_PORTE|GPIO_PIN6)
 
+/* ESP8266 TX control */
+#define WIFI_TX(_s)				stm32_gpiowrite(GPIO_8266_PD, _s);	/* enable/disable 8266 TX */
```

Comment:
@kd0aij , FYI @LorenzMeier 

ehmm... 
Please see this list of abstractons https://github.com/PX4/Firmware/blob/master/src/platforms/px4_micro_hal.h

It is px4_arch_xxxx that replaces stm32_xxxxx 

**But** when in the src/boards/xxxxx folders the code there uses the stm32_prefix because the board is arch specific **However** for published macros that will be used outside of src/boards the  px4_arch_ prefix is used